desktop/apps/auctions2/templates/index.jsx

```diff
@@ -1,35 +1,9 @@
-import PropTypes from 'prop-types'
 import React from 'react'
-import { withLayout } from 'desktop/components/react/main_layout'
-
-function Index (props) {
-  const {
-    Layout: { Header, Body, Footer },
-    Templates: { Meta }
-  } = props
 
+export default function Index (props) {
```

Comment:
Oh yeah, this will eventually get revised when i build it out. I'm partial to `IndexRoute` since its whats being mounted when navigating to `/auctions`, but I agree it's a bit vague.